hologram_dashboard/fetcher/testdata/hologram_config_malformed.config

```diff
@@ -0,0 +1,18 @@
+data_source_config {
```

Comment:
For such config files, we usually use .ascii as its extension.

Change the file name to hologram_config_malformed.ascii.